2stWeek/pirit.jeong/6.js

```diff
@@ -0,0 +1,88 @@
+const getRipeTomatoes = (board) => {
+  const ripeTomatoes = [];
+  for (let i = 0; i < board.length; i++) {
+    for (let j = 0; j < board[0].length; j++) {
+      if (board[i][j] === 1) {
+        ripeTomatoes.push([i, j]);
+      }
+    }
+  }
+
+  return ripeTomatoes;
+};
+
+const solution = (n, m, board) => {
+  const q = getRipeTomatoes(board);
+  const visited = new Array(n).fill(false).map(() => new Array(m).fill(false));
+
+  const directions = [
+    [0, -1],
+    [0, 1],
+    [-1, 0],
+    [1, 0],
+  ];
+
+  while (q.length !== 0) {
+    const [x, y] = q.shift();
```

Comment:
이렇게 shift를 사용해서 시간초과가 안나셨나요??
전 아래같이 바꿨습니다!
```suggestion 
    const [x, y] = q[idx]
```